cinema/views.py

```diff
@@ -34,6 +38,10 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
```

Comment:
Did you use django debug toolbar? We have two many-to-many fields here ;)

Reply:
and ? I use different query sets in actions 
<img width="1324" alt="Screenshot 2022-11-26 at 16 07 06" src="https://user-images.githubusercontent.com/96727778/204092838-97a0b534-8ba3-4666-b5c4-774364e312d1.png">
<img width="1480" alt="Screenshot 2022-11-26 at 16 07 14" src="https://user-images.githubusercontent.com/96727778/204092840-e9aa6c85-7e0a-4dda-b3ac-fe81de582f9e.png">

Reply:
Thank you)